Packages/Application/Neos.NeosIo.Event/Resources/Private/Templates/TypoScriptObjects/FooterContent.html

```diff
@@ -11,14 +11,25 @@ <h3 class="h1 typo3-neos-alignment-center">Organizer</h3>
 		</f:if>
 		<div class="gi u-w1/1 u-wm6/12 u-wl6/12">
 			<f:if condition="{organizers.properties.logo}">
-				<figure>
-					<media:image asset="{organizers.properties.logo}" title="{organizers.properties.title}" alt="{organizers.properties.title}" maximumWidth="700" maximumHeight="500" allowUpScaling="false" />
-				</figure>
+				<f:if condition="{organizers.properties.website}">
```

Comment:
I just talked to Christopher and as far as I understood the caching thing this shouldn't be a problem like, Gerhard wrote.
However I would plan to refactor the footer and use an own prototype to render the organizer data. But we need to merge this before to be able to launch the website finally.